modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -81,26 +80,22 @@ export type CreateProposalBaseParams = {
 };
 
 export interface IVoteProposalParams {
-  pluginAddress: string;
   vote: VoteValues;
-  proposalId: number;
+  proposalId: string;
 }
 
 export interface IExecuteProposalParams {
-  pluginAddress: string;
-  proposalId: number;
+  proposalId: string;
 }
 
 export interface ICanVoteParams {
-  pluginAddress: string;
-  proposalId: number;
+  proposalId: string;
   address: string;
-  vote: VoteValues
+  vote: VoteValues;
 }
 
 export type CanExecuteParams = {
```

Comment:
As said above, we can keep this type or make it redundant.
Is it likely that new params would be needed in the future?